### PR TITLE
Removing top level nesting

### LINKED ISSUE #57: [Proposal] Remove unnecessary nesting at top level
## What is your proposed change?

To remove unnecessary nesting at the top level to make the JSON schema easier to read. 

## Why is this not covered by the existing model?

The current model is  

```

"title": "Risk Data Library Schema",
    "type": "object",
    "required": [...],
    "properties": {
   	 "common": {
   		 "type": "object",
   		 "properties": {
   			 "contribution": {
   				 "type": "object",
   				 "required": [...],
   				 "properties": {
   					 "title": {

```

Suggesting this is changed to 

```
"title": "Risk Data Library Schema",
    "type": "object",
    "required": [...],
    "properties": {
   	 "title": {
```

### PR DESCRIPTION
**Pull request by odscrachel**
Closes #57. Please note this is the first of many structural changes, so the json is still not valid

### CODE REVIEW

**Comment by odscrachel** [comment]
Once #92 has been merged, need to update the changelog

File: schema/rdl_schema_0.1.json [hunk 1]
```diff
@@ -4,123 +4,130 @@
...
+		"maintainer": {
+			"type": "string",
+			"description": "Who added the resource"
+		},
+		"maintainer_email": {
+			"type": "string",
+			"description": "Contact email"
+		}
+	},
+	"resources": {
```
Comment (duncandewhurst) [inline schema/rdl_schema_0.1.json:96]: `resources` should be under `properties`.

**Review by duncandewhurst — CHANGES_REQUESTED** [review]

File: schema/rdl_schema_0.1.json (hunk 1 above)
Comment (duncandewhurst) [reply schema/rdl_schema_0.1.json]: Fixed in https://github.com/GFDRR/rdl-standard/pull/93/commits/60cd76ca7bc2205f3de29177d018ea9e4ca5580c

**Comment by duncandewhurst** [comment]
@odscjen please can you review my changes in https://github.com/GFDRR/rdl-standard/pull/93/commits/60cd76ca7bc2205f3de29177d018ea9e4ca5580c?

**Review by odscjen — DISMISSED** [review]
The nesting from the `common` object have all been removed, but the unnecessary top level nesting within the `vulnerability`, `loss` and `exposure` objects is still there. That'll be taken care of in a subsequent PR.

In terms of this PR removing unnecessary nesting in the top level object this can be approved.

**Review by duncandewhurst — DISMISSED** [review]

**Comment by duncandewhurst** [comment]
Let's wait for https://github.com/GFDRR/rdl-standard/pull/92 to be merged then we can merge `dev` into this branch and add a changelog entry before merging this PR.

**Comment by odscjen** [comment]
merged in the changelog as Stu had approved, and updated to include a line about this change. No idea why checks are failing, error is 

"/home/runner/work/rdl-standard/rdl-standard/docs/about/history/changelog.md" is not formatted.

**Comment by duncandewhurst** [comment]
How to resolve check failures is documented in https://github.com/GFDRR/rdl-standard/blob/dev/developer_docs.md#resolve-check-failures, specifically:

> #### mdformat
> 
> If this check fails, run the following command to fix markdown formatting:
> 
> ```bash
> mdformat docs
> ```

I've done that so the checks now pass.

**Review by duncandewhurst — APPROVED** [review]

**Review by stufraser1 — APPROVED** [review]

**Pull request merged**